Program.cs

```diff
@@ -1,24 +1,94 @@
-﻿using System;
-using System.Collections.Generic;
-using System.IO;
-using System.Linq;
-using System.Threading.Tasks;
-using Microsoft.AspNetCore;
-using Microsoft.AspNetCore.Hosting;
-using Microsoft.Extensions.Configuration;
-using Microsoft.Extensions.Logging;
-
-namespace PhoneEdit
+using Microsoft.AspNetCore.Identity;
+using Microsoft.EntityFrameworkCore;
+using MySqlConnector;
+using PhoneEdit.Data;
+
+var builder = WebApplication.CreateBuilder(args);
+
+try
+{
+    var identityContextSecrets = builder.Configuration["ConnectionStrings:IdentityContext"] ?? 
+                                 throw new InvalidOperationException("Connection string 'IdentityContext' not found.");
+    var phoneBookContextSecrets = builder.Configuration["ConnectionStrings:PhoneBookContext"] ?? 
+                                  throw new InvalidOperationException("Connection string 'PhoneBookContext' not found.");
+        
+    builder.Services.AddDbContext<ApplicationDbContext>(options => 
+        options.UseSqlite(identityContextSecrets));
+    
+    builder.Services.AddDbContext<PhonebookContext>(options => 
+        options.UseMySql(phoneBookContextSecrets, ServerVersion.AutoDetect(phoneBookContextSecrets))
+            .LogTo(Console.WriteLine, LogLevel.Information)
+            .EnableSensitiveDataLogging()
+            .EnableDetailedErrors()
+        );
+}
+catch (MySqlException ex)
+{
+    throw new InvalidOperationException("Error: ", ex);
```

Comment:
I changed the catch block so that the original MySqlException is thrown directly.

```cs
catch (MySqlException ex)
{
    throw ex;
}
```

[Commit](https://github.com/rumkit/PhoneEdit/pull/8/commits/ba3fead2fe9ef0646b27428b7e77fae6b36423d8)